src/cpp/src/llm_pipeline.cpp

```diff
@@ -65,17 +67,46 @@ ov::genai::OptionalGenerationConfig get_config_from_map(const ov::AnyMap& config
         return std::nullopt;
 }
 
+void remove_special_tokens(ov::genai::TokenizedInputs& tokenized_input,
```

Comment:
luckily this bug is solved in openvino_tokenizers, so `remove_special_tokens` can be safely dropped.

Reply:
yep, removed it, thx!